### DISCUSSION TITLE
1.0 Release Date Projection

### DISCUSSION THREAD

**Discussion by grantjayy** [vlang/v#21427] [General]
I love V and want to use it beyond simple hobby projects. However, I’d like it to be more stable and feature complete before building an enterprise application. 

I understand that designing a language is difficult, and projecting a 1.0 release date is arguably harder.

But the thing is… I’m ecstatic to bring this language to my team and want to plan for the future!

Does the core team have an estimate for a 1.0 release candidate?

**Comment by JalonSolov** [comment]
"When it's ready."

No release dates.
